devise.rb

```diff
@@ -136,14 +123,24 @@
   run 'rm app/controllers/application_controller.rb'
   file 'app/controllers/application_controller.rb', <<~RUBY
     class ApplicationController < ActionController::Base
-    #{  "protect_from_forgery with: :exception\n" if Rails.version < "5.2"}  before_action :authenticate_user!
+      before_action :authenticate_user!
     end
   RUBY
 
   # migrate + devise views
   ########################################
   rails_command 'db:migrate'
   generate('devise:views')
+  gsub_file(
+    'app/views/devise/registrations/new.html.erb',
+    '<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>',
+    '<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name), data: { turbo: :false }) do |f| %>'
+  )
+  gsub_file(
+    'app/views/devise/sessions/new.html.erb',
+    '<%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>',
+    '<%= simple_form_for(resource, as: resource_name, url: session_path(resource_name), data: { turbo: :false }) do |f| %>'
+  )
```

Comment:
Do we want to keep turbo at all? Otherwise we could even uninstall `gem "turbo-rails"` and package `@hotwired/turbo-rails` maybe?

Reply:
As we saw together this is fixed when using node >`16` like we do in the setup 👏